cmd/oras/pull.go

```diff
@@ -161,14 +181,10 @@ func runPull(opts pullOptions) error {
 			// named content downloaded
 			pulledEmpty = false
 		}
+		printed.LoadOrStore(display.Name(desc), true)
```

Comment:
What is the purpose of this line? Why the returned values are ignored?

Reply:
Should use https://pkg.go.dev/sync#Map.Store here

Reply:
Changed to `sync.Map.Store`